docs/man/docker-run.1.md

```diff
@@ -230,10 +229,6 @@ which interface and port to use.
 **--lxc-conf**=[]
    (lxc exec-driver only) Add custom lxc options --lxc-conf="lxc.cgroup.cpuset.cpus = 0,1"
 
-**--log-driver**="|*json-file*|*syslog*|*none*"
```

Comment:
I don't think so. I'm using `json-log` and `syslog` pretty regular with `docker run`.